sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_models.py

```diff
@@ -48,11 +48,70 @@ class SettingSelector:
     :keyword label_filter: A filter to select configuration settings based on their labels. Default is value is
      EMPTY_LABEL i.e. (No Label) as seen in the portal.
     :type label_filter: Optional[str]
+    :keyword tag_filters: A filter to select configuration settings based on their tags. This is a list of strings
+     that will be used to match tags on the configuration settings. Reserved characters (\\*, \\, ,) must be escaped
+     with backslash if they are part of the value.
+    :type tag_filters: Optional[List[str]]
     """
 
-    def __init__(self, *, key_filter: str, label_filter: Optional[str] = EMPTY_LABEL):
+    def __init__(
+        self, *, key_filter: str, label_filter: Optional[str] = EMPTY_LABEL, tag_filters: Optional[List[str]] = None
+    ):
+        if tag_filters is not None:
+            if not isinstance(tag_filters, list):
+                raise TypeError("tag_filters must be a list of strings.")
+            if len(tag_filters) > 5:
```

Comment:
This check not needed since we dont do the same check for comma separated keys either. Let server fail if more than 5 filters are given